### ISSUE TITLE
Issue: opening an llm run in playground is unreliable and breaks often

### ISSUE THREAD

**Issue by ldorigo** [langchain-ai/langsmith-sdk#513]
### Issue you'd like to raise.

Example run (private... so not sure if you can use it): https://smith.langchain.com/o/048a3fde-58b6-48a3-a4e5-cfaece013c46/playground/r/95edece3-6903-4ffb-8a03-eaf33c7d1441

Problems:

- Sometimes it just works
- Sometimes it just hangs forever
- Sometimes I get a random error (see screencast below)

[langsmith_bug.webm](https://github.com/langchain-ai/langsmith-sdk/assets/8631181/54cc6c68-fa11-4fe3-8b41-613772fa0ea7)

**Comment by hinthornw** [comment]
How is the LLM run being logged? Is it a LangChain object? Traceable decorated function ?

**Comment by ldorigo** [comment]
It's a langchain AzureChatOpenAI object invoked as part of a Langgraph graph

**Comment by hinthornw** [comment]
Gotcha! Flagged it to the playground lead. Working on a resolution

**Comment by ldorigo** [comment]
Hi thanks for the high level of responsiveness. I found out that the hanging error was caused by a problem in my azure deployment (and in Langchain which doesn't seem to raise Azure API errors? but that's not directly for you). However, I keep getting the other type of errors:

![image](https://github.com/langchain-ai/langsmith-sdk/assets/8631181/c1d4fc3b-a545-4c74-b315-7bccdca3a8cf)


This is for this run: https://smith.langchain.com/o/048a3fde-58b6-48a3-a4e5-cfaece013c46/playground/r/cbb8d943-89c2-4216-87fd-f7d23d49acef
